### ISSUE TITLE
How to determine the version of the twbx file?

### ISSUE THREAD

**Issue by pjcpjc** [tableau/document-api-python#189]
Is there anything in a .twbx file to determine which version it is? I'm working on a system whereby the .twbx gets deployed in a cloud environment and the .twbx file needs to be "saved as" an older version of tableau in order to run online. 
I don't think `tableaudocumentapi` could be helpful in terms of being able to change the "saved as" version of the file, but it would be nice if my `tableaudocumentapi` script could at least print out some diagnostic information in case the deployment engineer forgot to perform this step.

**Comment by pjcpjc** [comment]
Not sure how active this repo is, but @RussTheAerialist  appears to responding to things....

**Comment by jacalata** [comment]
You're right that we couldn't do the actual upgrade, but version info is in the twbx and we could make it available through this 

It's an attribute on the workbook element: 
<workbook source-build='9.3.0 (9300.16.0315.0125)' source-platform='win' version='9.3' xml:base='https://tableau.laatgetallenspreken.nl' xmlns:user='http://www.tableausoftware.com/xml/user'>
